Packs/cisco-meraki/Integrations/cisco-meraki/cisco-meraki.yml

```diff
@@ -622,6 +623,13 @@ script:
       description: Organization id
     - name: headers
       description: Table's headers to be shown by order
+    outputs:
+    - contextPath: status
+      description: license status
+      type: String
+    - contextPath: expirationDate
+      description: license expiration date
+      type: String
```

Comment:
This was missing? Did you catch this using the SDK command?

Reply:
It was missing. I caught this after executing this command manually and saw that in the readme file the output file of the command is empty.